examples/model_selection/plot_validation_curve.py

```diff
@@ -18,53 +18,25 @@
 
 from sklearn.datasets import load_digits
 from sklearn.svm import SVC
-from sklearn.model_selection import validation_curve
+from sklearn.model_selection import ValidationCurveDisplay
 
 X, y = load_digits(return_X_y=True)
 subset_mask = np.isin(y, [1, 2])  # binary classification: 1 vs 2
 X, y = X[subset_mask], y[subset_mask]
 
-param_range = np.logspace(-6, -1, 5)
-train_scores, test_scores = validation_curve(
+param_name, param_range = "gamma", np.logspace(-6, -1, 5)
+disp = ValidationCurveDisplay.from_estimator(
     SVC(),
     X,
     y,
-    param_name="gamma",
+    param_name=param_name,
     param_range=param_range,
-    scoring="accuracy",
+    score_type="both",
+    log_scale=True,
     n_jobs=2,
+    score_name="Accuracy",
 )
```

Comment:
On the rendered example, there's no fill between for the training curve. Is it there but the std is very close to 0 ?

Reply:
The std. dev. is really close to zero. With a `make_classification`, I get the following:

<img width="581" alt="image" src="https://user-images.githubusercontent.com/7454015/228291505-22625be8-fcd6-4a28-acca-cd426678b6a9.png">